html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -44,7 +44,7 @@ The captival portal block
 
 has_block 'captive_portal' =>
   (
-    render_list => [qw(logo redirecturl always_use_redirecturl nbregpages sms_pin_retry_limit)],
+    render_list => [qw(logo redirecturl always_use_redirecturl nbregpages sms_pin_retry_limit sms_request_retries)],
```

Comment:
How about `sms_request_limit`? It would adopt the format of the new `sms_pin_retry_limit` parameter.